languages/go/example/example.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	sdk "github.com/bitwarden/sdk/languages/go"
+	"github.com/gofrs/uuid"
+)
+
+func main() {
+	apiURL := os.Getenv("API_URL")
+	identityURL := os.Getenv("IDENTITY_URL")
+	userAgent := os.Getenv("USER_AGENT")
+
+	bitwardenClient := sdk.NewBitwardenClient(&apiURL, &identityURL, &userAgent)
+
+	accessToken := os.Getenv("ACCESS_TOKEN")
+	organizationIDStr := os.Getenv("ORGANIZATION_ID")
+	projectName := os.Getenv("PROJECT_NAME")
+
+	if projectName == "" {
+		projectName = "NewTestProject" // default value
+	}
+
+	responseForAPIKeyLoginResponse := bitwardenClient.AccessTokenLogin(accessToken)
+	fmt.Println(responseForAPIKeyLoginResponse)
+
+	organizationID, err := uuid.FromString(organizationIDStr)
+	if err != nil {
+		panic(err)
+	}
+
+	projectResponse, err := bitwardenClient.Projects.Create(organizationID.String(), projectName)
```

Comment:
The variable names in the example should be updated to reflect that the responses are models not response models.

Reply:
done